### DISCUSSION TITLE
Ingestion issue in Minio

### DISCUSSION THREAD

**Discussion by ankit289** [Sunbird-cQube/community#850] [General]
When we try to ingest any new program data in minio we are not getting any error and data is also not showing in minio.
@RNarmatha As discussed in meeting we have checked the data files, spec and grammar all Ok, but still not getting file on minio.

**Comment by RNarmatha** [comment]
Hi @ankit289 ,
Thanks for the update. Could you please share the those files with Rajesh 
@bommalla - Could you please take it up

**Reply by ankit289** [reply]
ok

**Comment by bommalla** [comment]
@ankit289 @RNarmatha 

Just now I checked your event and schema found some issues.Please find the below points

1. Date format should be dd/mm/yy format but in CSV file dd-mm-yy is there
2. In schema date column is there in middle  but your csv file date is there first
3. Please check event csv and based on that you can prepare the schema(order is impotent)

Please check the above points and re-ingest again

**Comment by bommalla** [comment]
@ankit289 @RNarmatha 

This is not code issue . It's completely data issues bro.. 

Please check properly before ingesting.

**Reply by ankit289** [reply]
@bommalla I have shared updated files to you let me know if these files are according to the template or not

**Comment by bommalla** [comment]
@ankit289 @RNarmatha 

yes it's correct please check below points as well
1. check event data with schema 
2. check dimension for lo-wise and total-lo-wise
3. check inside programs/pat please check event grammar for lo-wise and total_lo_wise
